themes/finna2/templates/RecordDriver/DefaultRecord/record-audio-player.phtml

```diff
@@ -1,4 +1,7 @@
 <!-- START of: finna - RecordDriver/DefaultRecord/record-audio-player.phtml -->
+<?php
+$this->headScript()->appendFile('vendor/video.min.js');
```

Comment:
videojs:ää ei ladata suoraan sen koon takia, vaan initVideoPopup tarkistaa scripts-attribuutin ja lataa dynaamisesti. Näin huolehditaan myös siitä, ettei tule montaa erilaista js-bundlea.

Reply:
Muutettu käyttämään `finna.layout.loadScripts`:iä.